Sources/Interaction/Style/InteractorStyleRemoteMouse/index.js

```diff
@@ -120,12 +120,23 @@ function vtkInteractorStyleRemoteMouse(publicAPI, model) {
 
   //-------------------------------------------------------------------------
   publicAPI.handleMouseWheel = (callData) => {
-    publicAPI.invokeRemoteWheelEvent({
-      type: 'MouseWheel',
-      ...createRemoteEvent(callData),
-      spinY: callData.spinY,
-    });
-    publicAPI.invokeInteractionEvent(INTERACTION_EVENT);
+    let needToSend = true;
+    if (model.wheelThrottleDelay) {
```

Comment:
Why bother checking `wheelThrottleDelay` ?

If it is 0, then `needToSend = model.whe...` will always be equal to `true`, isn't it ?

Reply:
good point. I added the check just to completely isolate the new code (so the default code path wouldn't really be modified), but I will update. Thanks for the feedback!

Reply:
oh, sorry, looking at the code again, the only benefit to the check is that when wheelThrottleDelay isn't set, Date.now() won't get called (which is slightly expensive)- should I update anyway?